coordinator/README.md

```diff
@@ -3,6 +3,23 @@
 Want to build Stargate locally or even start contributing to the project? This is the right place to get started. 
 
 If you're developing on macOS, we've added notes throughout to highlight a few specific differences.
+
+## Quickstart
+
+The fastest way to build your own copy of the Stargate coordinator code and Docker images involves the following steps:
+
+* Local build of coordinator images
+  * Make sure you are in the `coordinator` directory and have `JAVA_HOME` set to point to a JDK 1.8 installation
```

Comment:
Maybe emphasize that later JDK versions do not work, too (as reader might otherwise assume it's just minimum needed).